packages/ndla-tabs/src/FilterTabs.tsx

```diff
@@ -56,16 +92,20 @@ class FilterTabs extends Component {
   }
 
   updateTabSizes() {
-    if (!this.tabWidths && this.liRefs[Object.keys(this.liRefs)[0]].parentNode.offsetWidth !== 0) {
+    const firstLi = this.liRefs[Object.keys(this.liRefs)[0]];
+    //@ts-ignore No way to know that parentNode has offsetWidth
+    if (!this.tabWidths && firstLi?.parentNode?.offsetWidth !== 0) {
```

Comment:
Er det mulig å gjøre en typesjekk på parentNode som gjør at man kan være sikker på at offsetWidth finnes?

Reply:
Ser ut som `parentElement` er et `HTMLElement` som har `offsetWidth` kan vi bruke den?
Evt så kan vi caste (eller lage en litt funky typeguard for `is HTMLElement`)?

Reply:
Endret fra `parentNode` til `parentElement`. De ga samme ouput, og sluttresultatet så ut til å fungere likt.